customerio/RemoteHabits-Android
# feat: add in-app event listener

Reopening PR #57 

### QA Instructions:

* Install `PR.61` builds for testing
* To test that the event listeners are working, you will use Remote Habits logs (not SDK logs). If you see log statements such as "in-app message shown. message ...." then the event listener received an event successfully. 
* Additionally, you will also see events on dashboard for every call received in handler, the event will look like:
  * _Activity Type:_ Event
  * _Activity Name:_ In-App Event 
  * _Attributes:_
    * **event_name:** messageDismissed
    * **delivery_id:** deliveryID
    * **message_id:** in-app-test-message
* You should see log messages for the following events (you will need to make these events happen in the app for in-app messages):
  * message shown
  * message dismissed
  * message error happened (maybe you can't test this?)
  * message action taken
